BUILDING.md

```diff
@@ -160,17 +189,25 @@ installed to be able to compile the C/C++ sources.
 > cmake --build .
 ```
 
-Using gRPC C++ as a DLL is not recommended, but you can still enable it by running `cmake` with `-DBUILD_SHARED_LIBS=ON`.
+### Windows Build Notes 
 
-### Windows: A note on building shared libs (DLLs)
+**Building shared libs (DLLs)**
 
-Windows DLL build is supported at a "best effort" basis and we don't recommend using gRPC C++ as a DLL as there are some known drawbacks around how C++ DLLs work on Windows. For example, there is no stable C++ ABI and you can't safely allocate memory in one DLL, and free it in another etc.
+> **Warning**
+> Using gRPC C++ as a DLL is not recommended, but you can still enable it by running `cmake` with `-DBUILD_SHARED_LIBS=ON`. 
+
+Windows DLL build is supported on a "best effort" basis and we don't recommend using gRPC C++ as a DLL as there are some known drawbacks around how C++ DLLs work on Windows. For example, there is no stable C++ ABI and you can't safely allocate memory in one DLL, and free it in another etc.
```

Comment:
I would recommend removing the `"best effort" basis` clause, because it has not been supported in years to my knowledge, and I'm not aware of anyone who's willing to take that work up. It is plainly not recommended to use.

Reply:
The diff makes it look like I've added this text but it was already present, word-for-word. 

All I've actually tried to do here is rationalize the warning relating to using gRPC DLLs (there were two instances) and combine it with the explanation that expands the point. I've not modified content here. 


BTW, I note that installing gRPC using `vcpkg` installs a DLL build. (I discovered that the hard way.)

I have no objection to the message being updated and clarified. I'm just not sure I'm best-placed to make the point.

Reply:
> I have no objection to the message being updated and clarified. I'm just not sure I'm best-placed to make the point.

I'm in a pretty good position to make that point, I'd appreciate if you'd remove that clause for me. If not I can do it afterwards, but it should be removed one way or another. Thanks.

Reply:
As requested, I've removed the clauses and clarified that Windows DLL build is not recommended.